src/Configuration/Config.AzureKeyVault/ref/Microsoft.Extensions.Configuration.AzureKeyVault.netstandard2.0.cs

```diff
@@ -16,6 +17,17 @@ public static partial class AzureKeyVaultConfigurationExtensions
 }
 namespace Microsoft.Extensions.Configuration.AzureKeyVault
 {
+    public partial class AzureKeyVaultConfigurationOptions
+    {
+        public AzureKeyVaultConfigurationOptions() { }
```

Comment:
Do you need the no-paramater constructor? I'd think it would cause people to not discover the others.

Reply:
I'd like to leave the ability to set everything manually without going through any custom logic